src/main/java/com/dsec/backend/model/repo/RepoUpdateDTO.java

```diff
@@ -0,0 +1,45 @@
+package com.dsec.backend.model.repo;
+
+import com.dsec.backend.entity.RepoDomain;
+import com.dsec.backend.entity.RepoType;
+import com.fasterxml.jackson.annotation.JsonAlias;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+@NoArgsConstructor
+public class RepoUpdateDTO {
+
+    @JsonAlias("full_name")
+    @NotEmpty(message = "Please enter a the repository full name, in the format username/repository-name")
+    private @NonNull String fullName;
+
+    @JsonAlias("repo_name")
+    @NotEmpty(message = "Please enter a repository name")
+    private @NonNull String repoName;
+
+    @JsonAlias("description")
```

Comment:
JsonAlias is used when deserializing info from JSON. This was used when fetching the data from Github because they use this representation of fields. You should not add this JsonAlias to DTOs that are used to communicate with our frontend.